docs/manual/introduction.tex

```diff
@@ -833,6 +833,28 @@
 Do not check the JRE version the Checker Framework is running on.
 \end{itemize}
 
+Compatibility options
+\begin{itemize}
+
+\item \<-AaliasedTypeAnnos=\emph{aliases}>
+  Defines custom type annotation aliases for the canonical annotations of a checker.
+  `aliases` is in the format
+
+`FQN.canonical.Qualifier1:FQN.alias1.Qual1,FQN.alias2.Qual1;FQN.canonical.Qualifier2:FQN.alias1.Qual2`.
+
+This can only be used for annotations without attributes, e.g. this will not be usable to declare an alias for @Regex(2).
+When in a Unix system, `;` need to be escaped.
+
+\item \<-AaliasedDeclAnnos=\emph{aliases}>
+  Defines custom declaration annotation aliases for the canonical annotations of a checker.
+  `aliases` is in the format
+
+`FQN.canonical.Qualifier1:FQN.alias1.Qual1,FQN.alias2.Qual1;FQN.canonical.Qualifier2:FQN.alias1.Qual2`.
+
+This can only be used for annotations without attributes, e.g. this will not be usable to declare an alias for @Regex(2).
```

Comment:
```suggestion
This can only be used for annotations without attributes, e.g. this will not be usable to declare an alias for `@EnsuresNonNull(...)`.
```